google/cloud/bigtable_v2/services/bigtable/client.py

```diff
@@ -380,6 +382,9 @@ def __init__(
             transport (Union[str, BigtableTransport]): The
                 transport to use. If set to None, a transport is chosen
                 automatically.
+                NOTE: "rest" transport functionality is currently in a
+                beta state (preview). We welcome your feedback via an
+                issue in this library's source repository.
```

Comment:
Is there any way to opt out of rest for the data client?

Reply:
Yes, I believe it's configured in the yaml somewhere. We can address at that in a future PR

Reply:
@igorbernstein2 Can you say more about that? We have gotten some customer feedback in other product areas that REST transport is important to them in some use cases, so we've typically preferred to include the option when we can and let users choose which they require with sensible defaults. Are you suggesting we disallow it entirely? (I see the previous cl/458022604 where this was disallowed, but I'm wondering if those are still relevant.)

Reply:
@meredithslota the rest transport doesn't support asyncio, so I think it makes sense to hard-code the transport to use our pooled transport for at least the async layer.

When it comes time to add the synchronous layer, we could support rest if customers find it useful, but it just wouldn't be able to make use of the same channel management optimizations that we're adding to grpc side. Maybe @igorbernstein2 has some more context though